<launch>
    <node name="velocity_controller_node" pkg="RC" type="velocity_controller_node" output="screen">
        <param name="kP" value="0.0001"/>
        <param name="kI" value="0.000000001"/>
        <param name="kD" value="0.01"/>
        <param name="sensor_topic" value="encoder_vel"/>
    </node>
</launch>
